libraries/lwIP_Ethernet/src/LwipIntfDev.h

```diff
@@ -399,7 +399,7 @@ bool LwipIntfDev<RawDev>::begin(const uint8_t* macAddress, const uint16_t mtu) {
         if (RawDev::interruptIsPossible()) {
             noInterrupts(); // Ensure this is atomically set up
             pinMode(_intrPin, INPUT);
-            attachInterruptParam(_intrPin, _irq, LOW, (void*)this);
+            attachInterruptParam(_intrPin, _irq, HIGH, (void*)this);
```

Comment:
Only comment I have would be to make this a function from the template specialization so you can be `high` here and the other Ethernet devices can be `low`.

Did you get any idea how much this helps with latency for, say, `ping`?